arch/platform/openmote/openmote-b/board.h

```diff
@@ -190,7 +176,7 @@
  * @{
  */
 #define ANTENNA_BSP_RADIO_BASE   GPIO_PORT_TO_BASE(GPIO_D_NUM)
-#define ANTENNA_BSP_RADIO_INT    GPIO_PIN_MASK(3)
+#define ANTENNA_BSP_RADIO_INT    GPIO_PIN_MASK(4)
 #define ANTENNA_BSP_RADIO_EXT    GPIO_PIN_MASK(4)
```

Comment:
Why the change of the pin number?

Reply:
This change also looks weird to me, but 2.4-GHz communication becomes much more reliable as a result. I just tested again with CSL on a fixed channel/no retransmissions/maximum length frames. The old configuration reaches 68% successful transmissions, whereas the new one reaches 99.8%.

Reply:
Could you create a separate PR with your antenna fixes? Then, I will remove mine from here.

Reply:
#2684